ecommerce/android/Getflix/app/src/main/java/com/example/getflix/activities/MainActivity.kt

```diff
@@ -36,7 +29,7 @@ class MainActivity : AppCompatActivity() {
         var name = ""
         var sproducts = null
         val scategories = null
-        const val BASE_URL = "http://10.0.2.2:8000/"
+        const val BASE_URL = "http://ec2-18-189-28-20.us-east-2.compute.amazonaws.com:8000/"
```

Comment:
I think you should move this variable into the class that proposes API requests since it is not related to the main activity.

Reply:
moved to GetflixAPIService now.